project/BuildDependencies/scripts/0_package.native-win32.list

```diff
@@ -7,7 +7,7 @@
 ; -> ...
 ;PLEASE KEEP THIS LIST IN ALPHABETICAL ORDER!
 doxygen-1.8.2-win32.7z
-flatc-1.9.0-Win32-v141.7z
+flatc-1.11.0-win32-v141-20191222.7z
 jsonschemabuilder-1.0.0-win32-3.7z
 swig-3.0.10-win32.7z
 texturepacker-1.1.1-win32.7z
```

Comment:
newer version available

Reply:
it's not part of kodi-deps so there's no reason for me to touch that now

Reply:
It's part of depends and as Windows doesn't build dependencies from source that one has to rebuild and bundled too.